opencodelists/tests/fixtures.py

```diff
@@ -0,0 +1,289 @@
+"""This module contains fixtures to be used in all apps.
+
+One of the design goals of the project is that there should be a limited and controlled
+number of states that objects can be ing  This is achieved by requiring (through
```

Comment:
`ing` -> `in.`

Reply:
Fixed in f90032fe285ed70a9b4e821773d90046a0231876.